src/main.rs

```diff
@@ -42,6 +44,9 @@ struct Cli {
     /// Log SSL pre-master key to file in SSLKEYLOGFILE environment variable.
     #[clap(long)]
     keylog: bool,
+    /// metrics server address
```

Comment:
"Bind address on which to serve Prometheus metrics" or something else more descriptive?